tokens/utils.js

```diff
@@ -162,7 +162,37 @@ async function transformInPath(location, variablesMap, transformType = 'definiti
   }
 }
 
+
+function createIndexFile(buildDir, isTheme, themeVariant){
+  const directoryPath = isTheme ? `${buildDir}/themes/${themeVariant}`:`${buildDir}/core`;
+  
+fs.readdir(directoryPath, (err, files) => {
+  if (err) {
+    console.error('Error reading directory:', err);
+    return;
+  }
+
+  const jsonFiles = files.filter(file => file !== 'index.css');
+  isTheme && jsonFiles.reverse();
```

Comment:
It may be worth leaving a comment in the code explaining why the `.reverse()` is necessary.

Reply:
I add a comment, it's not exactly necessary, it's just to maintain the order, I mean variables first when run build-scss.

A more explicit approach could be the following:

```javascript
outputCssFiles.sort((a,b) => a.includes('variables') ? -1 : 0)
```